modules/apps/default.nix

```diff
@@ -3,6 +3,6 @@
 {
   imports = [
     ./konsole.nix
-    ./kate.nix
+	./kate
```

Comment:
Weird indenting here (run `nixpkgs-fmt` if you use this, or manually edit if you prefer)

Reply:
I wasn't aware of `nixpkgs-fmt`, I ran it now on the file.